pkg/kubelet/prober/worker.go

```diff
@@ -46,6 +45,9 @@ type worker struct {
 	// Describes the probe configuration (read-only)
 	spec *api.Probe
 
+	// The type of the worker.
+	probeType probeType
+
```

Comment:
I am not against keeping probeType here, but would like to limit its use to debugging if possible. Adding fields seems cleaner to me because a worker is not going to change its probeType half way. I am ok with the current approach but would prefer adding fields. I wouldn't block the PR on this though.